intel_extension_for_transformers/llm/operator/cscr/dispatcher/src/jblas_weightonly_dispatcher.cpp

```diff
@@ -491,19 +522,24 @@ void parse_gemm_core_offline(qbits_config_param* p, qbits_runtime_ctx* ctx) {
                             jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight,
                             jblas::gemm::GemmCore_Row_NN_8x48_AVX512_VNNI, jblas::utils::parallel::Parallel2DGemm,
                             JblasAVX512_VNNI>(p, ctx);
-      TORCH_CHECK(false,
-                  "Qbits: device ISA must lagger than AVX512_VNNI when "
-                  "GemmCore==Row_NN_8x48_AVX512_VNNI");
+      TORCH_CHECK(false, "Qbits: device ISA must lagger than AVX512_VNNI when GemmCore==Row_NN_8x48_AVX512_VNNI");
+      break;
+    case jblas::gemm::GemmCoreType::AVX_VNNI_2x48:
+      assert(p->compute_type == "int8");
+      if (check_avx_vnni())
+        return parse_weight<TASK, jblas::wrapper::gemm_pack_weight::GemmInterfaceParallelAB,
+                            jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight,
+                            jblas::gemm::GemmCore_Row_NN_2x48_AVX_VNNI, jblas::utils::parallel::Parallel2DGemm,
+                            JblasAVX_VNNI>(p, ctx);
+      TORCH_CHECK(false, "Qbits: device ISA must lagger than AVX_VNNI when GemmCore==Row_NN_2x48_AVX_VNNI");
```

Comment:
```large``` is an internal definition.
```suggestion
      TORCH_CHECK(false, "Qbits: device ISA must support AVX_VNNI when GemmCore==Row_NN_2x48_AVX_VNNI");
```